engine/virt/virt.go

```diff
@@ -211,7 +207,7 @@ func (v *Virt) VirtualizationInspect(ctx context.Context, ID string) (*enginetyp
 		return nil, err
 	}
 
-	bytes, err := json.Marshal(coretypes.LabelMeta{Publish: []string{"PORT"}})
+	marshal, err := json.Marshal(coretypes.LabelMeta{Publish: []string{"PORT"}})
```

Comment:
why change this?

Reply:
bytes collides with imported package bytes.